java/src/jmri/jmrix/lenz/messageformatters/XNetMessageFormattersBundle_nl.properties

```diff
@@ -0,0 +1,150 @@
+
+XNetReplyLIVersion                          = LI10x Hardware-versie: {0} Software-versie: {1}
+XNetReplyLIBaud                             = ANTWOORD LI101 Baudsnelheid: {0}
+XNetReplyLIAddress                          = ANTWOORD LI101 Adres {0}
+XNetReplyCSStatus                           = Status Centrale:
+
+# XNetReply Command Station Status Bytes.  These are concatenated together 
+# in the code.
+XNetCSStatusEmergencyOff                    = Noodstop/Emergency Uit
+XNetCSStatusEmergencyStop                   = Noodstop/Emergency Stop
+XNetCSStatusServiceMode                     = Service-modus
+XNetCSStatusPoweringUp                      = Inschakelen (Power-up) bezig
+XNetCSStatusPowerModeAuto                   = Auto power-up stand
+XNetCSStatusPowerModeManual                 = Handmatige power-up stand
+XNetCSStatusRamCheck                        = RAM-Controle fout!
+XNetCSStatusRamNormal                       = Normale werking
+
+# XNetReply Loco information
+XNetReplyLocoStatus13Label                  = Loc F13-F28 Tijdelijk-status:
+XNetReplyLocoNormalLabel                    = Loc-informatie antwoord: Normal tractie
+XNetReplyLocoMULabel                        = Loc-informatie antwoord: Loc in Multitractie
+XNetReplyLocoMUBaseLabel                    = Loc-information antwoord: Multitractie basisadres
+XNetReplyLocoDHLabel                        = Loc-informatie antwoord: Loc in Dubbeltractie
+XNetReplyLoco2DHLabel                       = Tweede Loc in Dubbeltractie is:
+XNetReplyLocoLabel                          = Loc-informatie Reactie:
+XNetReplySearchNormalLabel                  = Zoek-antwoord: Normale Loc:
+XNetReplySearchDHLabel                      = Zoek-antwoord: Loc in Dubbeltractie:
+XNetReplySearchMUBaseLabel                  = Zoek-antwoord: MT-basisadres:
+XNetReplySearchMULabel                      = Zoek-antwoord: Loc in MT:
+XNetReplySearchFailedLabel                  = Zoek-antwoord: Zoeken mislukt naar:
+XNetReplyLocoOperated                       = wordt bediend door een ander apparaat.
+XNetReplyFStatusLabel                       = Functie-status:
+XNetReplyF13StatusLabel                     = F13-F28 status:
+XNetReplyFeedbackLabel                      = Feedback-antwoord:
+TurnoutWFeedback                            = Wissel met terugkoppeling
+TurnoutWoFeedback                           = Wissel zonder terugkoppeling
+XNetReplyNotOperated                        = Niet bediend
+XNetReplyThrownLeft                         = Linksaf
+XNetReplyThrownRight                        = Rechtsaf
+XNetReplyInvalid                            = <ongeldig>
+XNetReplyFeedbackEncoder                    = Feedback Encoder Basis-adres:
+XNetReplyContactLabel                       = Contact:
+XNetReplyAddressInUse                       = Adres in gebruik door ander apparaat.
+XNetReplyAddressFree                        = Adres beschikbaar voor instructies.
```

Comment:
This file doesn't have an empty line at the end of the file.

Reply:
fixed